ansible/server-state-playbooks/nightshade-web/requirements.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+- name: openmicroscopy.omero-common
+  src:  https://github.com/openmicroscopy/ansible-role-omero-common.git
```

Comment:
Alternatively since you've got git installed on the machine that's running Ansible you can just add `version: X.X.X`, the main advantage of https://github.com/openmicroscopy/pydoop-features/commit/fec18e1 is you don't need to install git.